### DISCUSSION TITLE
Standup (Wednesday, May 4, 2022)

### DISCUSSION THREAD

**Discussion by antonieto** [CarlosCRG19/ijalti-peers#38] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

``` 
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by paotg123** [comment]
**What did you achieve in the last 24 hours?**:
 - I saw a course

**What are your priorities for the next 24 hours?**:
 - finish it

**Blockers**:
 -
**Shoutouts**:
 - @DavBot02 @CristianCazares

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Initialized package, and written our first ORM entity with @BrandonMagana 
 - Fixed a typescript compiler bug

**What are your priorities for the next 24 hours?**:
 - Read more about TypeORM
 - Read more about auth0 and other alternatives for authentiation

**Blockers**:
 - Poor knowledge about TypeORM

**Shoutouts**:
 - @DavBot02 , @CristianCazares , @paotg123 for working so hard on learning so many technologies so quickly. Keep it up viejones 🔥.

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on figma wireframes
 - Read about github actions

**What are your priorities for the next 24 hours?**:
 - Continue to develop wireframes on Figma
 - Maybe help the backend team to get TypeORM set up

**Blockers**:
 - wtf with Figma's UX

**Shoutouts**:
 - @DavBot02 , @CristianCazares , @paotg123 for their discipline and dedication to learning a new technology

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished PERN course
 - Read more about Axios and backend connection

**What are your priorities for the next 24 hours?**:
 - Read more about Material UI

**Blockers**:
 - 

**Shoutouts**:
 - @paotg123 @CristianCazares for keeping up the good work

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Still learning about React and Express primarily.

**What are your priorities for the next 24 hours?**:
 - Focus more on Material UI.
 - Help with anything that I can do.

**Blockers**:
 - There are soooo many routes to take.

**Shoutouts**:
 - @antonieto @BrandonMagana @CarlosCRG19 for working on the sprint stuff while us mortals are doing courses.

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Create RM, and code along @antonieto our first entity

**What are your priorities for the next 24 hours?**:
 -  Read TypeORM documentation
 -  Create first endpoints

**Blockers**:
 -  Haven't worked with typeorm before

**Shoutouts**:
 - @antonieto for fixing the typescript bug, @CristianCazares @DavBot02 @paotg123  @CarlosCRG19  for being working hard
